examples/BUILD.bazel

```diff
@@ -113,6 +113,10 @@ example_integration_test(
 
 example_integration_test(
     name = "examples_jest",
+    bazel_commands = ["build ..."],
```

Comment:
this looks wrong, we won't run `bazel test` on the example anymore, so how do we know if it works?

Reply:
Oh, copy paste :)

Didn't really understand how the integration tests work